apis/flowcollector/v1beta2/flowcollector_types.go

```diff
@@ -175,14 +175,15 @@ type FlowCollectorIPFIX struct {
 // - `DNSTracking`, to track specific information on DNS traffic.<br>
 // - `FlowRTT`, to track TCP latency.<br>
 // - `NetworkEvents`, to track network events [Developer Preview].<br>
-// +kubebuilder:validation:Enum:="PacketDrop";"DNSTracking";"FlowRTT";"NetworkEvents"
+// +kubebuilder:validation:Enum:="PacketDrop";"DNSTracking";"FlowRTT";"NetworkEvents";"EbpfManager"
 type AgentFeature string
 
 const (
 	PacketDrop    AgentFeature = "PacketDrop"
 	DNSTracking   AgentFeature = "DNSTracking"
 	FlowRTT       AgentFeature = "FlowRTT"
 	NetworkEvents AgentFeature = "NetworkEvents"
+	EbpfManager   AgentFeature = "EbpfManager"
```

Comment:
I wonder if `bpfman` deserve more visibility here. Maybe have a dedicated section to be able to configure extra parameters such as `BPFMAN_BPF_FS_PATH` ?

Also, is the final plan to keep the ability to be privileged while using bpfman ? These two options seems very tied to me

Reply:
the limitation with privileged when we use bpfman is the map type used by pca the final goal was to move away from perf map to use ringbuf and that will remove this limitation and improve performance when pca feature is enabled